pkg/gateway/wallet.go

```diff
@@ -6,12 +6,78 @@ SPDX-License-Identifier: Apache-2.0
 
 package gateway
 
-// A Wallet stores identity information used to connect to a Hyperledger Fabric network.
-// Instances are created using factory methods on the implementing objects.
-type Wallet interface {
-	Put(label string, id IdentityType) error
-	Get(label string) (IdentityType, error)
+import (
+	"encoding/json"
+
+	"github.com/pkg/errors"
+)
+
+type wallet interface {
+	Put(label string, id Identity) error
+	Get(label string) (Identity, error)
 	Remove(label string) error
 	Exists(label string) bool
-	List() []string
+	List() ([]string, error)
+}
+
+// A Wallet stores identity information used to connect to a Hyperledger Fabric network.
+// Instances are created using factory methods on the implementing objects.
+type Wallet struct {
+	store WalletStore
+}
+
+// Put an identity into the wallet
+func (w *Wallet) Put(label string, id Identity) error {
+	content, err := id.toJSON()
+	if err != nil {
+		return err
+	}
+
+	return w.store.Put(label, content)
+}
+
+// Get an identity from the wallet. The implementation class of the identity object will vary depending on its type.
+func (w *Wallet) Get(label string) (Identity, error) {
+	content, err := w.store.Get(label)
+
+	if err != nil {
+		return nil, err
```

Comment:
After using this sdk for a long time I can say that it would be good to start having error types, not just plain text

Reply:
Hello, I also want to join this team to develop fabric-sdk-go. How to join and work together with this team? Besides, to be honest, updates of  fabric-sdk-go are to slowly to satisfy practical needs.